### ISSUE TITLE
New release fail the docker stack test on arm64, something wrong with pip install pymatgen

### ISSUE THREAD

**Issue by unkcpz** [aiidalab/aiidalab-qe#723]
https://github.com/aiidalab/aiidalab-docker-stack/actions/runs/9006100849/job/24745209326 
The arm64 test failed because of new release of `aiidalab-qe==24.4.0`. Seems pymatgen not able to be installed from pip. @superstar54 @AndresOrtegaGuerrero can you have a look at it? It block the release of docker image with `aiida-core==2.5.1`

**Comment by superstar54** [comment]
I don't have arm64, maybe @AndresOrtegaGuerrero could test this. @unkcpz, could you provide a short description of how to test this?

**Comment by AndresOrtegaGuerrero** [comment]
I think is related to this https://github.com/mikibonacci/aiidalab-qe-vibroscopy/issues/52 , to be able to install aiidalab-qe-vibroscopy i had to access the docker as root user to be able to install h5py . During my update to aiida 2.5 i was able to install h5py

**Comment by unkcpz** [comment]
To reproduce the issue, I think you can run in an arm64 with the `full-stack:edge` and install the latest version of qeapp (`v24.4.0`) from appstore. From the error message of failed integration test, it seems a dependency related to `pymatgen`. My laptop do not has enough resource to run the docker now, so it has to be @AndresOrtegaGuerrero or @mikibonacci can test locally. But I think you can reproduce it. 

For the fix, sorry, the dependencies issues are quite nasty, you have to go to libraries and compare to check which one might cause the problem.

**Comment by AndresOrtegaGuerrero** [comment]
@unkcpz Yes, but also had to install and update , pkg-config , in the processs. my aiida got updated to 2.5 and then i was able to install h5py (as root user )

**Comment by AndresOrtegaGuerrero** [comment]
@unkcpz I just did the test with edge image, i manage to install the qe app , but i cant install the vibroscopy plugin. 
`Failed to build h5py
ERROR: Could not build wheels for h5py, which is required to install pyproject.toml-based projects` , the app also shows to incompatible.
